model_compression_toolkit/core/pytorch/back2framework/mixed_precision_model_builder.py

```diff
@@ -143,14 +143,15 @@ def _get_weights_configurable_quantizer_kwargs(self, n: BaseNode, attr: str) ->
                 'max_candidate_idx': max_candidate_idx
                 }
 
-    def mixed_precision_activation_holder(self, n: BaseNode) -> PytorchActivationQuantizationHolder:
+    def mixed_precision_activation_holder(self, n: BaseNode, holder_type=PytorchActivationQuantizationHolder) -> PytorchActivationQuantizationHolder:
         """
         Retrieve a PytorchActivationQuantizationHolder layer to use for activation quantization for a node.
         The layer should hold either a configurable activation quantizer, if it is quantized with mixed precision,
         or an inferable quantizer for fixed single bit-width quantization.
 
         Args:
             n: Node to get PytorchActivationQuantizationHolder to attach in its output.
+            holder_type: ActivationQuantizationHolder type
```

Comment:
please change description to: "The type of the activation quantization holder to use."